internal/gamepad/gamepad.go

```diff
@@ -187,11 +187,30 @@ type Gamepad struct {
 	native nativeGamepad
 }
 
+type mappingKind int
+
+const (
+	noMapping mappingKind = iota
+	axisMapping
+	buttonAnalogMapping
+	buttonDigitalMapping
+	buttonMixedMapping
+	hatUpMapping
+	hatRightMapping
+	hatDownMapping
+	hatLeftMapping
+)
```

Comment:
I prefer using `mapping` as a prefix rather than `mappingKind` as a suffix

```
mappingKindNone
mappingKindAxis
mappingKindButtonAnalog
...
```

Reply:
Done.